### PR TITLE
Fix audio length

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#39 
#### What does this implement/fix? Explain your changes.
Below audio files have been looped to have length between `1.5`s - `2.5`s:
- `3.wav`
- `4.wav`
- `5.wav`
- `6.wav`
- `7.wav`
- `8.wav`
- `9.wav`
- `10.wav`
- `11.wav`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- alarms' length
```
Comment (sepandhaghighi) [inline CHANGELOG.md:9]: - alarms' length modified

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Well done 🚀 

Please take a look at my comments and also:

- Update `TONES.md` (**duration** and **size** sections)

**Comment by sadrasabouri** [comment]
It's now ready for your review @sepandhaghighi

**Review by sepandhaghighi — APPROVED** [review]
🚀

**Pull request merged**